devops/helm/clamAV/_clamav/configurations/1.0/clamd.conf

```diff
@@ -547,7 +547,7 @@ MaxScanSize 150M
 # Technical design limitations prevent ClamAV from scanning files greater than
 # 2 GB at this time.
 # Default: 100M
-MaxFileSize 30M
+MaxFileSize 4M
```

Comment:
It is well known the file will be increased to 15 MB. I do not see a reason to not leave this ready since there is no harm if we even never do it.